infra-in-containers/youngee.md

```diff
@@ -0,0 +1,103 @@
+# Docker
+Docker is a program that create, deploy and run applications by utilising container. Developers can package a application with all the required libraries and dependencies in a container which can be virtually run anywhere.
+
+## Install Docker CE
+To use Docker, let's do an [installation for Docker CE (Community Edition)](https://docs.docker.com/install/linux/docker-ce/debian/#set-up-the-repository). Follow the instructions from the link and Docker CE should be now installed.
+
+Test it out by running `docker run hello-world`
+The output of the command should show:
+```
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+...
+```
+
+## Common Docker Commands
```

Comment:
These are good commands to learn by hard. You guys will be using them a lot in the days to come.